zebra-state/src/service.rs

```diff
@@ -156,7 +156,7 @@ pub(crate) struct StateService {
     /// - the hash of the last valid committed block (the parent of the invalid block).
     //
     // TODO:
```

Comment:
I think this TODO was implemented as `SentHashes`, which is just below this field.

```suggestion
```

Reply:
We should be able to get rid of this field as part of https://github.com/ZcashFoundation/zebra/issues/6388.